src/components/templates/ProductTemplate/ProductTemplateProgress/ProductTemplateProgress.md

```diff
@@ -0,0 +1,3 @@
+### Summary
+
+`<ProductTemplateProgress>` the _<Progress />_ component with fixed styling.
```

Comment:
adding `_<Progress />_` adds a funky progress bar in the docs
![Screenshot 2020-09-21 at 11 14 38](https://user-images.githubusercontent.com/1081070/93755719-aa417100-fbfb-11ea-9b5e-788b2cd982e6.png)

Reply:
Fixed. Thanks!